### PR TITLE
Readme m1

### PR DESCRIPTION
**Pull request by jn2clark**
* **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [X] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
adds a section for getting good performance using M1 macs with Marqo

* **What is the current behavior?** (You can also link to an open issue here)
no instructions

* **What is the new behavior (if this is a feature change)?**
instructions

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,12 +203,51 @@ Searching using an image can be achieved by providing the image link.
...
+6. Install marqo dependencies,
+```
+pip install -e .
+```
+7. Change into the tensor search directory,
+```
+CWD=$(pwd)
+cd src/marqo/tensor_search/
+```
+8. Run Marqo;
```
Comment (pandu-k) [inline README.md]: strange semicolon here

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Tensor search uses deep-learning to transform documents, images and other data i
 ## Getting started
 
 1. Marqo requires docker. To install docker go to https://docs.docker.com/get-docker/
-2. Use docker to run Marqo:
+2. Use docker to run Marqo (M1 users will need to [go here](#M1-Mac-users)):
```
Comment (pandu-k) [inline README.md]: It may not be clear what M1 means:  perhaps "Mac M1 chip users  ...

**Review by pandu-k — CHANGES_REQUESTED** [review]
Created two comments to be looked into. Besides this, all good.

File: README.md (hunk 1 above)
Comment (jn2clark) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (jn2clark) [reply README.md]: done

**Pull request merged**